src/main/java/com/fundation/webservice/model/Metadata.java

```diff
@@ -41,8 +41,8 @@ public void writeXmpFile(File file) {
     }
 
     /**
-     * This method writes a json file containing the param file metadata info returned by exiftool in JSON format.
-     * The json file is stored in the same directory where the param file is located.
+     * Writes a json file containing the param file metadata info returned by exiftool in JSON format.
+     * Stores the json file in the same directory where the param file is located.
```

Comment:
This might not be in accordance to the team's adopted redaction style.

AFAIR, it should somthing like: Method that stores...

Reply:
resolved

Reply:
Now, it is compliant, Good job!